docs/HLL/Hll_vs_CS_Hllpp.md

```diff
@@ -226,7 +226,7 @@ Depending on the chosen configuration, the *HllSketch* can be from one to almost
 ****
 
 * [1] [DataSketches HllSketch GitHub](https://github.com/apache/datasketches-java/tree/master/src/main/java/org/apache/datasketches/hll)
-* [2] [DataSketches HllSketch JavaDocs](/api/java/snapshot/apidocs/index.html)
+* [2] [DataSketches HllSketch JavaDocs (top of page)](https://datasketches.apache.org/docs/Background/TheChallenge.html)
```

Comment:
I don't understand this change

Reply:
The original link is clearly broken because we changed the way we access the javadocs.  
The natural replacement for that link would be 
`https://apache.github.io/datasketches-java/6.0.0/org/apache/datasketches/hll/package-summary.html` 
But notice that it refers to a specific release 6.0.0, which would be a maintenance nightmare.  I replaced it at too high of a level.  I should have just pointed it to https://datasketches.apache.org/docs/HLL/HLL.html

Will fix.